Tests/kaas/k8s-default-storage-class/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
If I understood the ADR right the directory `Tests/kaas/k8s-default-storage-class` should contain an example for `_Option 3_ Write Python scripts for tests`, but I see here a bash script that IMO does not contain such an example.

Reply:
Isn't the `run_checks.sh` the file "starting" a test or did I misread either your comment or the file?

Reply:
yep, you are right I overlooked the run_checks.sh script.